quiche/src/ffi.rs

```diff
@@ -743,7 +743,7 @@ pub extern fn quiche_conn_send(
             out_info.from_len = std_addr_to_c(&info.from, &mut out_info.from);
             out_info.to_len = std_addr_to_c(&info.to, &mut out_info.to);
 
-            std_time_to_c(&info.at, &mut out_info.at);
+            out_info.in_nanos = info.at.duration_since(time::Instant::now()).as_nanos() as u64;
```

Comment:
We could also just calculate the nanos (as you did here) and fill the `timespec` struct by hand... This would at least not break the API. WDYT ?

Reply:
most likely @ghedo has some opinions here :)

Reply:
I thought about this as well. I decided against this because it seems unnecessary to construct that timespec to just bridge over. We also would need to get the current time spec and then add the calculated nanos. IMO, that is all not worth the hassle and just returning nanos as we do with the timeout APIs works. Happy to hear more opinions on this!

Reply:
Yeah nanos are fine for me... Just wanted to bring up the suggestion :)